user_interface/src/common/konva/drawLayer.js

```diff
@@ -107,11 +107,11 @@ export const addKonvaDrawLayer = (stageNode) => {
       listening: false,
     });
 
-    // Do not add if the DrawRect is too small (Less than 1 pixel)
-    if (drawRect.width() > 1 && drawRect.height() > 1) {
+    // Do not add if the DrawRect is too small (Less than 3 pixel)
```

Comment:
```suggestion
    // Do not add if the DrawRect is too small (Less than 3 pixels)
```

Reply:
I think this check should be moved up, before creating newRect. If not there'll be wasted work